.vscodeignore

```diff
@@ -2,4 +2,10 @@
 node_modules
 src/
 tsconfig.json
-webpack.config.js
\ No newline at end of file
+webpack.config.js
+gulpfile.js
+.editorconfig
+CONTRIBUTING.md
+Jenkinsfile
+*.jenkins
+images/
```

Comment:
We will likely need to include `images/` in the `.vsix`, since these images are linked in the `README.md`. The `README.md` is used in the extension marketplace. There are differences between https://marketplace.visualstudio.com/items?itemName=redhat.vscode-xml and the `README.md` from the repo. This means the marketplace is likely using the `README.md` and images that are bundled in the `.vsix` instead of from GitHub. This means if we don't bundle the images, then they won't appear on the marketplace.

Reply:
the images/ folder at the root only contains images used by contributing.md. 
Readme uses URLs to the GH repo. 

I tried the embedded doc after installing the vsix and it worked (but you should check it anyway)

Reply:
Okay. Sorry about that.